### DISCUSSION TITLE
DSC Panel Erroring with Keybus fault

### DISCUSSION THREAD

**Discussion by hluetkens** [Dilbert66/esphome-dsckeybus#122] [Q&A]
Just setting up using this configuration which is provided. As soon as I hook the ESP32 card to my system. My control keypads put a warning beep and state "keybus fault, contact service". Not sure if there is something I need to do to my control keypads? I may have missed something within the setup instructions for this. Really not sure.

**Comment by hluetkens** [comment]
Found duplicate on other discussion

**Comment by Dilbert66** [comment]
Could be many things. Make sure zone expanders are set to zero, check your wiring, check the addresses of ports, etc.
